README.md

```diff
@@ -408,6 +410,32 @@ Example:
       thing'
 ```
 
+### Action: `append`
+
+Appends a string on the end of a given file. File must already exist. If no
+newline at end of `with` parameter, one will be added unless
+`skip_ensure_newline` is set to `true`.
+
+If you need to remove an existing trailing newline before appending, use
+`regex_replace` instead.
+
+Params:
+- `paths`: List of files and/or directory trees to append to end of.
```

Comment:
I'm a little worried about accepting multiple values as an array instead of an array of a concrete type:

```yaml
paths: ['a', 'b']
```

versus

```yaml
paths:
  - path: 'a'
  - path: 'b'
```

The second is more verbose, but gives us flexibility in the future to add control values in the future:

```yaml
paths:
  - path: 'a'
    banana: 'apple'
  - path: 'b'
```

Reply:
This was done to be consistent with all the existing actions which use paths as a list. If we want to change that, it should probably be a separate issue that changes all known templates and actions at once. (Assuming we are not guaranteeing a stable API yet)

Reply:
https://github.com/abcxyz/abc/issues/130